lib/assesments/assessment_2/fi_ass2_rohan/profile/fi_ass2_rohan_profile.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi_ass2_rohan/Colors/fi_ass_rohan_colors.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi_ass2_rohan/profile/widgets/fi_ass_rohan.part3.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi_ass2_rohan/profile/widgets/fi_ass_rohan_part1.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi_ass2_rohan/profile/widgets/fi_ass_rohan_part2.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi_ass2_rohan/profile/widgets/fi_ass_rohan_part4.dart';
+
+import '../Widgets/SizedBox/fi_ass2_rohan_size.dart';
+
+class FiAssRohanProfile extends StatelessWidget {
+  const FiAssRohanProfile({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+          backgroundColor: Colors.white,
+          appBar: AppBar(
+            backgroundColor: Colors.white,
+            elevation: 0.0,
+            leading: IconButton(
+                onPressed: () {
+                  Navigator.pop(context);
+                },
+                icon: const Icon(
+                  Icons.arrow_back,
+                  color: MyColors.black,
+                )),
+            title: const Text(
+              "My Profile",
+              style:
+                  TextStyle(fontWeight: FontWeight.bold, color: MyColors.black),
+            ),
+            centerTitle: true,
+            actions: const [
+              Padding(
+                padding: EdgeInsets.only(right: 10),
+                child: Icon(
+                  Icons.settings,
+                  color: MyColors.black,
+                ),
+              ),
+            ],
+          ),
+          body: Column(
+            children: [
+              const FiAssRohanPart1(),
```

Comment:
Please use better names that define what the widget is, I can't really understand what part1 is